data_inconsistencies.py

```diff
@@ -32,17 +32,35 @@ def get_inconsistencies():
     return incons
 
 
+def print_statistics(file_name, n_incons, firefox, chrome):
+    f = utils.get_all_images()
+    total_img = len(f)
+    incons_f = [int(x) for x in firefox]
+    incons_c = [int(x) for x in chrome]
+    print("Number of Photos: {} " .format(total_img))
```

Comment:
Nit: uppercase only for the first letter and names (in this case `Number of photos`)